### PR TITLE
Add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by rffontenelle**

### CODE REVIEW

**Comment by rffontenelle** [comment]
I'm not seeing my translation in the flatpak artifact produced by the GitHub Actions run. I even used `strace` to look for a call to the machine-readable translation file (.mo), but none was found.  Shouldn't it be showing up translated?

**Comment by aleiepure** [comment]
Flatpak handles translations as a separate extension of the main executable. The GitHub Action doesn't produce it yet. [See here](https://github.com/flatpak/flatpak-github-actions/issues/46#issue-985431383).

**Comment by aleiepure** [comment]
I've updated the manifest to include translation as part of the main bundle. Should work now. Let me know.

**Comment by rffontenelle** [comment]
Weird. The artifact still doesn't show up in Portuguese and there is build error message `Failed to save cache: ReserveCacheError: Unable to reserve cache with key flatpak-builder-7d8318a1884e1f88600126c3573868366a20c86e-x86_64, another job may be creating this cache.`

But that's not important right now. If it works once released, it is good for me.

**Comment by aleiepure** [comment]
It's because it was run before my change to the workflow. I've already tested it with the other two translations and it works fine.

**Comment by rffontenelle** [comment]
Awesome, thanks!

File: po/pt_BR.po [hunk 1]
```diff
@@ -0,0 +1,1027 @@
+# Portuguese translations for devtoolbox package
+# Traduções em português brasileiro para o pacote devtoolbox.
+# Copyright (C) 2023 THE devtoolbox'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the devtoolbox package.
+# Rafael Fontenelle <rafaelff@gnome.org>, 2023.
```
Comment (aleiepure) [inline po/pt_BR.po:5]: At the moment this file is not REUSE compliant. You have to add the SPDX-License line with any license to pass compliance:
```suggestion
# Rafael Fontenelle <rafaelff@gnome.org>, 2023.
# SPDX-License-Identifier: GPL-3.0-or-later
```

**Review by aleiepure — CHANGES_REQUESTED** [review]

File: po/pt_BR.po (hunk 1 above)
Comment (rffontenelle) [reply po/pt_BR.po:5]: Please consider this information as instruction for those willing to contribute with translation. Personally I'm not use to have this requirement.

Comment (aleiepure) [reply po/pt_BR.po:5]: It's already written in the translation instructions but I'll make it more obvious.

**Review by aleiepure — APPROVED** [review]

File: po/pt_BR.po (hunk 1 above)
Comment (rffontenelle) [reply po/pt_BR.po:5]: ah, totally missed it. Sorry.

**Pull request merged**